src/components/shared/DiagramCard.tsx

```diff
@@ -36,7 +36,11 @@ const DiagramCard: React.FC<DiagramCardProps> = ({
             className={classNames(container, className)}
         >
             {imageData && (
-                <GatsbyImage image={imageData} alt={cardTitle || "diagram"} />
+                <GatsbyImage
+                    style={{ marginBottom: 16 }}
```

Comment:
i could remove this: some of the images are cropped real close on the bottom, it would be nice if they were standardized